### PR TITLE
More SDQL power.

### PR DESCRIPTION
**Pull request by PJB3005**
IT'S HAPPENING.

https://dl.dropboxusercontent.com/u/107028026/ShareX/2016/04/2016-04-09_00-55-49.webm

Arbitrary SDQL function calls, list accessing.

### CODE REVIEW

File: code/modules/admin/verbs/SDQL_2/SDQL_2_parser.dm [hunk 1]
```diff
@@ -354,6 +354,14 @@
 		L += "."
 		i = variable(i + 2, L)
 
+	else if (token(i + 1) == "(") // OH BOY PROC
+		to_chat(world, "FOUND PROC:")
```
Comment (Tastyfish) [inline code/modules/admin/verbs/SDQL_2/SDQL_2_parser.dm]: Debugging code

File: code/modules/admin/verbs/SDQL_2/SDQL_2_parser.dm [hunk 2]
```diff
@@ -354,6 +354,14 @@
...
+		var/list/arguments = list()
+		i = call_function(i, null, arguments)
+		L += ":"
+		L[++L.len] = arguments
+		to_chat(world, list2json(L))
```
Comment (Tastyfish) [inline code/modules/admin/verbs/SDQL_2/SDQL_2_parser.dm]: see above

File: code/modules/admin/verbs/SDQL_2/SDQL_2_parser.dm (hunk 1 above)
Comment (PJB3005) [inline code/modules/admin/verbs/SDQL_2/SDQL_2_parser.dm]: Yep i know, I'm still gonna add to this.

**Comment by gbasood** [comment]
DNM because debug code

**Pull request merged**